spec/models/spree/user_spec.rb

```diff
@@ -201,4 +201,38 @@
       expect(user.flipper_id).to eq "Spree::User;42"
     end
   end
+
+  describe "#disabled=" do
+    context 'when the value is truthy' do
+      it "sets disabled datetime" do
+        user = create(:user)
```

Comment:
You don't need to store the user in the database. You could use `build(:user)` for an unsaved record built with test data or simple Ruby because you don't need any other attributes:
```suggestion
        user = Spree::User.new
```